linker/src/main/kotlin/com/openlattice/linking/matching/SocratesMatcher.kt

```diff
@@ -50,10 +52,14 @@ class SocratesMatcher(
         private val linkingFeedbackService: PostgresLinkingFeedbackService
 ) : Matcher {
 
-    private var localModel = ThreadLocal.withInitial { model.clone() }
+    private val localModel = ParallelInference.Builder(model)
+                        .inferenceMode(InferenceMode.SEQUENTIAL)
+                        // .batchLimit(32)
```

Comment:
Need decision whether to do this sequentially or in batches.

Reply:
That doesn't sound very parallel.

Reply:
Batching makes the most sense here I think, just want to see if anyone has strong opinions/if I miss anything.